libbeat/scripts/generate_template.py

```diff
@@ -114,6 +116,16 @@ def fill_field_properties(field, defaults):
         properties[field["name"]] = {
             "type": "date"
         }
+    elif field.get("type") == "string" and field["name"] == "message":
```

Comment:
Instead of having  field.get("type") == "string" and field["name"] == "message if clause, I would change the filebeat template as following:

<pre>
- name: message
  type: string
  index: analyzed
  doc_values: true
  ignore_above: 0
</pre> 

that applies to the rule that I added below.